ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -95,52 +94,55 @@
     </F.HelperText>
   </Hds::Form::Checkbox::Field>
 
-  <form.input
-    @name='client_id'
-    @type='text'
+  <Hds::Form::TextInput::Field
+    name='client_id'
     @value={{@model.client_id}}
-    @label={{t 'form.client_id.label'}}
-    @helperText={{t 'form.client_id.help'}}
-    @error={{@model.errors.client_id}}
-    readonly={{false}}
-    as |field|
+    @isInvalid={{@model.errors.client_id}}
+    @isOptional={{true}}
+    disabled={{form.disabled}}
+    {{on 'input' (set-from-event @model 'client_id')}}
+    as |F|
   >
+    <F.Label>{{t 'form.client_id.label'}}</F.Label>
+    <F.HelperText>{{t 'form.client_id.help'}}</F.HelperText>
     {{#if @model.errors.client_id}}
-      <field.errors as |errors|>
+      <F.Error as |E|>
         {{#each @model.errors.client_id as |error|}}
-          <errors.message>{{error.message}}</errors.message>
+          <E.Message>{{error.message}}</E.Message>
         {{/each}}
-      </field.errors>
+      </F.Error>
     {{/if}}
-  </form.input>
+  </Hds::Form::TextInput::Field>
 
-  <form.input
-    @name='client_secret'
-    @type='password'
+  <Hds::Form::TextInput::Field
+    name='client_secret'
     @value={{@model.client_secret}}
-    @label={{t 'form.client_secret.label'}}
-    @helperText={{t 'form.client_secret.help'}}
-    @error={{@model.errors.client_secret}}
-    readonly={{false}}
-    @hasVisibilityToggle={{false}}
-    as |field|
+    @isInvalid={{@model.errors.client_secret}}
+    @isOptional={{true}}
+    disabled={{form.disabled}}
+    type='password'
```

Comment:
I think we still need the `@hasVisibilityToggle={{false}}`

Reply:
@ZedLi ahh good catch, fixed